packages/docsearch-react/src/DocSearch.tsx

```diff
@@ -214,6 +249,7 @@ export function DocSearch({ ...props }: DocSearchProps): JSX.Element {
             translations={props?.translations?.modal}
             isAskAiActive={isAskAiActive}
             canHandleAskAi={canHandleAskAi}
+            indexes={indexes}
```

Comment:
Why is DocSearch prop named `indices`, while subcomponents accept an `indexes` prop? Isn't that a typo? Why not always a consistent name?